studio/components/interfaces/Settings/General/Infrastructure/PauseProjectButton.tsx

```diff
@@ -49,8 +54,7 @@ const PauseProjectButton: FC<Props> = observer(({ projectRef, projectId }) => {
       })
       setLoading(false)
     } else {
-      app.onProjectPaused(projectId)
-      app.onProjectStatusUpdated(projectId, PROJECT_STATUS.GOING_DOWN)
+      await invalidateProjectsQuery(queryClient)
```

Comment:
same comment as per `ComputeInstanceSizePanel` just that this also applies for project pausing